django/contrib/postgres/search.py

```diff
@@ -300,3 +303,90 @@ class TrigramSimilarity(TrigramBase):
 class TrigramDistance(TrigramBase):
     function = ''
     arg_joiner = ' <-> '
+
+
+class LexemeCombinable(Expression):
+    BITAND = '&'
+    BITOR = '|'
+
+    def _combine(self, other, connector, reversed, node=None):
+        if not isinstance(other, LexemeCombinable):
+            raise TypeError(
+                'Lexeme can only be combined with other Lexemes, '
```

Comment:
"Lexeme" will need to be added to docs/spelling_wordlist.txt, but if we want to keep the diff a little smaller and not have to add "Lexemes", then how about:
```
'A Lexeme can only be combined with another Lexeme, '
```